### PR TITLE
feat: form elements - dropdown documentation

### PR DESCRIPTION
**Pull request by houbly**
- Added properties and examples for wmndsDropdown component
- added & updated params to reflect GDS
- moved component to new page

### CODE REVIEW

File: src/wmnds/components/form-elements/dropdown/_dropdown.njk [hunk 1]
```diff
@@ -2,22 +2,54 @@
...
-        contentText: 'Dropdown label',
+        contentText: params.label,
         for: id
-      }) | trim
+      })
     }}
-    <select class="wmnds-fe-dropdown__select" id="{{id}}" name="{{id}}">
-      <option value="" selected="selected">Choose from list</option>
-      {%- for i in range(1,6) %}
+    <select class="wmnds-fe-dropdown__select" id="{{id}}" name="{{name}}" {%if describedBy%}aria-describedBy="{{describedBy}}"{%endif%}>
```
Comment (catiarodriguescosta) [inline src/wmnds/components/form-elements/dropdown/_dropdown.njk:40]: Dropdown must have the individual errorclass not sure if there is one for select, but on input it is wmnds-fe-input--error. It should have a red border if there's an error.

**Review by catiarodriguescosta — CHANGES_REQUESTED** [review]
Left some comments! :)

File: src/wmnds/components/form-elements/dropdown/_dropdown.njk (hunk 1 above)
Comment (houbly) [reply src/wmnds/components/form-elements/dropdown/_dropdown.njk:40]: There isn't a class for it in the css and I checked zeplin but there are no error styles for dropdown so maybe its best to leave this for a separate issue?

**Review by catiarodriguescosta — APPROVED** [review]
Looks good! 👍

**Pull request merged**